### PR TITLE
Win 8 test

### PR DESCRIPTION
**Pull request by trwalke**
Fixes # .
Fix for https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2821

**Changes proposed in this request**
Reenabling test

**Testing**
Automated test

**Performance impact**
none

### CODE REVIEW

File: tests/Microsoft.Identity.Test.Integration.Win8/DeviceAuthenticationTests.cs [hunk 1]
```diff
@@ -23,7 +23,7 @@ public class DeviceAuthenticationTests
 
         
         [TestMethod]
-        [Ignore] // https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2821
+        //[Ignore] // https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2821
```
Comment (jmprieur) [inline tests/Microsoft.Identity.Test.Integration.Win8/DeviceAuthenticationTests.cs:26]: why not remove the line?

**Review by jmprieur — APPROVED** [review]
LGTM

File: tests/Microsoft.Identity.Test.Integration.Win8/DeviceAuthenticationTests.cs (hunk 1 above)
Comment (trwalke) [reply tests/Microsoft.Identity.Test.Integration.Win8/DeviceAuthenticationTests.cs:26]: ah, I removed the wrong line. will fix

**Review by pmaytak — APPROVED** [review]
Should this link #2821 instead?

**Comment by trwalke** [comment]
> Should this link #2821 instead?

Yes. thanks @pmaytak

**Review by bgavrilMS — APPROVED** [review]

**Review by SameerK-MSFT — APPROVED** [review]

**Pull request merged**